### ISSUE TITLE
Page shows no initial content in Safari

### ISSUE THREAD

**Issue by greystate** [umbraco/Umbraco.Marketplace.Issues#26]
It just dawned on me that having to choose a category before seeing anything is not the default experience - just mine, because I'm using Safari (v16.1 on macOS Ventura 13.0.1).

Looks like this:

<img width="1440" alt="Screenshot 2022-12-07 at 09 52 18" src="https://user-images.githubusercontent.com/5463/206132854-9e4c996b-1c80-45e5-99c7-9c9d6a4ebb8d.png">

Mind you, it was only after having seen the site in Firefox (while double-checking something else) that I saw this, so probably isn't a huge problem — as soon as I choose a category I get content. Also, clicking the logo to go back to the frontpage, I get the expected initial content of "Most popular" and "Recently added".

FWIW, here's the console error appearing on the initial load:

<img width="607" alt="Screenshot 2022-12-07 at 09 57 27" src="https://user-images.githubusercontent.com/5463/206133999-a145f110-4919-44fc-96aa-5ffcab3ea7d9.png">

**Comment by nathanwoulfe** [comment]
Hey @greystate from what I can see, this is related somehow to minification and how Safari parses the minified JS - the uninitialized variable in this case is the homepage component itself, which would obviously be causing some major issues everywhere if it weren't browser-specific.. Will try to track down the cause, but may need to find an HQer on a Mac to investigate properly.

**Comment by greystate** [comment]
A somewhat worse consequence of this JS-requirement: Clicking the **List my package** button in the footer does absolutely nothing... or maybe even worse: Even typing in the URL `https://marketplace.umbraco.com/listing` still just gives you an empty page.

This one doesn't throw any console errors, but maybe there's been some updates already?

Still Safari only, yes, but that includes iPhones, so...

**Comment by greystate** [comment]
Please let me know how I can help fix this, as it's driving me absolutely crazy 😬

**Comment by greystate** [comment]
Looks like the frontpage issue has been fixed - should I report a separate issue for the /listing page?

**Comment by AndyButland** [comment]
It's OK thanks @greystate - we'll pick it up from here.

**Comment by greystate** [comment]
Awesome @AndyButland — can confirm this has been fixed 👍 (both `/` and `/listing`)

**Comment by AndyButland** [comment]
Great, good to hear.  Thanks for reporting back.

**Comment by nathanwoulfe** [comment]
In the interests of full transparency, nothing was changed explicitly related to this issue - something must have fixed it, but through no conscious effort on my part...
